packages/meteor/url_common.js

```diff
@@ -14,7 +14,7 @@ Meteor.absoluteUrl = function (path, options) {
     path = undefined;
   }
   // merge options with defaults
-  options = _.extend({}, Meteor.absoluteUrl.defaultOptions, options || {});
+  options = Object.assign({}, Meteor.absoluteUrl.defaultOptions, options || {});
```

Comment:
I think using `Object.assign` is safe here because the `es5-shim` package should already have had the chance to load by the time `Meteor.absoluteUrl` is called.